assets/dev/js/frontend/utils/lightbox/lightbox.js

```diff
@@ -225,7 +225,7 @@ module.exports = elementorModules.ViewModule.extend( {
 		if ( ! videoUrl ) {
 			const downloadImage = i18n.downloadImage;
 
-			$linkList.append( $( '<a>', { href: itemUrl, download: '' } ).text( downloadImage ).prepend( $( '<i>', { class: 'eicon-download-bold' } ) ) );
+			$linkList.append( $( '<a>', { href: itemUrl, download: '' } ).text( downloadImage ).prepend( $( '<i>', { class: 'eicon-download-bold', title: 'Download' } ) ) );
```

Comment:
The `title` attribute is visible on hover, and needs translation, please use `aria-label` here and all next.

Reply:
Done.